tools/flaky-test-reporter/main.go

```diff
@@ -126,7 +127,17 @@ func githubOperations(ghToken string, repoData []RepoData, dryrun bool) (map[str
 	return gih.processGithubIssues(repoData, dryrun)
 }
 
+func isWeekend(t time.Time) bool {
+	weekDay := t.Weekday()
+	return weekDay > 5 || weekDay < 1
```

Comment:
```suggestion
	return weekDay == 6 || weekDay == 0
```
This should look more intuitive.